README.md

```diff
@@ -106,7 +77,7 @@ select * from sample.result
 Run it with command:
 
 ```bash
-bash -c "$(python3 -m easy_sql.data_process -f sample_etl.spark.sql -p)"
+python3 -m easy_sql.data_process -f sample_etl.spark.sql
```

Comment:
This is not the recommended way to run etl with spark backend. Since some spark configuration is in the ETL file and will be parsed before run spark, and the task will likely to be submitted to a remote cluster.

Reply:
Get it, will revert it.